presto-openapi/src/main/resources/expressions.yaml

```diff
@@ -0,0 +1,232 @@
+          description: Optimized expression.
+        NativeSidecarFailureInfo:
+          $ref: "#/components/schemas/NativeSidecarFailureInfo"
+          description: Failure information when expression optimization throws an error.
+    NativeSidecarFailureInfo:
+      type: object
+      properties:
+        type:
+          type: string
+          description: Failure type.
+        message:
+          type: string
+          description: Error message for failure.
+        cause:
+          $ref: "#/components/schemas/NativeSidecarFailureInfo"
+          description: Cause of failure.
+        suppressed:
+          type: array
+          maxItems: 100
+          items:
+            $ref: "#/components/schemas/NativeSidecarFailureInfo"
+        stack:
+          type: string
+          description: Stack trace of failure.
+        errorCode:
+          $ref: "#/components/schemas/ErrorCode"
+          description: ErrorCode for the failure.
+    ErrorCode:
+      type: object
+      properties:
+        code:
+          type: integer
+          description: Numeric error code.
+        name:
+          type: string
+          description: String representation of the error.
+        type:
+          $ref: "#/components/schemas/ErrorType"
+          description: Indicates cause of error.
+        retriable:
+          type: boolean
+          description: Indicates whether the error could be resolved upon retry.
+    ErrorType:
+        type: string
+        enum:
+          - USER_ERROR
+          - INTERNAL_ERROR
+          - INSUFFICIENT_RESOURCES
+          - EXTERNAL
+    RowExpressions:
+      type: array
+      maxItems: 100
+      items:
+        $ref: "#/components/schemas/RowExpression"
+    RowExpression:
+      oneOf:
+        - $ref: "#/components/schemas/ConstantExpression"
+        - $ref: "#/components/schemas/VariableReferenceExpression"
+        - $ref: "#/components/schemas/InputReferenceExpression"
+        - $ref: "#/components/schemas/LambdaDefinitionExpression"
+        - $ref: "#/components/schemas/SpecialFormExpression"
+        - $ref: "#/components/schemas/CallExpression"
+    RowExpressionParent:
+      type: object
+      properties:
+        sourceLocation:
+          $ref: "#/components/schemas/SourceLocation"
+    SourceLocation:
+      description: The source location of the row expression in the original query, referencing the line and the column of the query.
+      type: object
+      properties:
+        line:
+          type: integer
+        column:
+          type: integer
+    ConstantExpression:
+      description: A constant expression is a row expression that represents a constant value.  The value attribute is the constant value.
+      allOf:
+        - $ref: "#/components/schemas/RowExpressionParent"
+        - type: object
+          properties:
+            "@type":
+              type: string
+              enum : ["constant"]
+            typeSignature:
+              type: string
```

Comment:
Can this be a complex type ?Would be great to clarify if there are any limitations about it.